docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+services:
+  veres-one:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    restart: unless-stopped
+    depends_on:
+      - mongo
+      - redis
+    ports:
+      - "127.0.0.1:42443:42443"
+    networks:
+      - app-network
+  mongo:
+    image: mongo:latest
```

Comment:
Do you have any thoughts about pinning to specific build?  Seems like if we lock all the base images down here, and have package lock files for our node code we would have a highly repeatable build.

Reply:
We don't need to pin to a specific build for development docker images. This was an attempt to make @aljones15 w/ developers being able to run a local docker-compose system. Fundamentally, I think this might have been a failed experiment, because the local docker setup is so different from the testing/production setups that it doesn't really help anything. If we  do want repeatable development builds, then yes, I have thoughts... having to do with using package-lock.json files, npm ci, etc... feels like the wrong solution for dev mode though. I'd rather just drop running docker locally entirely. Perhaps I should just delete this file entirely until we have a better solution.

Reply:
SUGGESTION: Delete docker-compose.yml file for dev mode until we have a better solution.

Reply:
OK, I did not know that this was in support of running docker-compose during dev.  So, the same question applies to our production systems.  If we don't somehow pin those docker images for node/mongo/redis those versions could slide from one of our releases to the next.

Reply:
FWIW, there seems to be a separate `docker-compose.yml` for production (which I don't think we should be doing)  that has images pinned to major versions https://github.com/veres-one/veres-one/pull/32/files/0fd385c32717a913e1938e2de95e6db29162228c#diff-90c20cfb41a25f76cc204d7ff294f43c

Reply:
> FWIW, there seems to be a separate docker-compose.yml for production (which I don't think we should be doing)

What do you think we should be doing instead?

Reply:
> If we don't somehow pin those docker images for node/mongo/redis those versions could slide from one of our releases to the next.

As @gannan08 mentioned, we do pin to major release versions in production and have the ability to pin to very specific hashes if necessary.

Reply:
@gannan08 and I talked about this, he'd like us to see if we can run a staging environment so devs can test out a fully integrated package. That process isn't written down anywhere, so we need to do that. But that work shouldn't hold up this PR.

Reply:
I have deleted the docker-compose.yml file for dev mode until we have a better solution.